docs/pages/kcard.vue

```diff
@@ -0,0 +1,359 @@
+          >
+            <template #title>
+              <div>
+                Kolibri video tutorials to understand the Learner, Coach, and Admin features.
+              </div>
+            </template>
+            <template #aboveTitle>
+              <div>
+                This is a sample above title slot text that can cover more that 3 lines
+              </div>
+            </template>
+            <template #footer>
+              <div>
+                This is a sample footer section for the kcard
+              </div>
+            </template>
+          </KCard>
+        </div>
+      </DocsShow>
+      <h3>
+        Horizontal Layout
+      </h3>
+      <p>
+        It's important to note that besides the mandatory   <DocsInternalLink href="/KCard#prop:title">
+          <code>title</code>
+        </DocsInternalLink> prop/slot, all other slots are optional. 
+        This flexibility allows for creating diverse layouts by combining different options for the <DocsInternalLink href="/KCard#prop:layout">
+          <code>layout</code>
+        </DocsInternalLink>
+        and
+        <DocsInternalLink href="/KCard#prop:thumbnailDisplay">
+          <code> thumbnailDisplay</code>
+        </DocsInternalLink>  properties. Additionally, the order of slots is not strictly enforced. For instance,
+        the  <DocsInternalLink href="/KCard#slot:title">
+          <code>title</code>
+        </DocsInternalLink> 
+        slot can take the position of the 
+        <DocsIternalLink href="/KCard#slot:aboveTitle">
+          <code>aboveTitle</code>
+        </DocsIternalLink> slot if the latter is not used.
+      </p>
+      <p>
+        The thumbnail's appearance depends on two factors: the size of the original image and the chosen
+        <DocsInternalLink href="/KImg#prop:thumnailcaletype">
+          <code>
+            thumbnailScaleType
+          </code>
+        </DocsInternalLink>
+        property.  This property controls how the image is scaled to fit the designated thumbnail area. 
+        The available scaling options are the same as those offered by the KImg component's scaleTypes property.
+      </p>
+      Here are a few examples...
+
+      <p>
+        Horizontal 
+        <DocsInternalLink href="/KCard#prop:layout">
+          <code> layout</code>
+        </DocsInternalLink> and 
+        <DocsInternalLink href="/KCard#prop:thumbnailDisplay">
+          <code> thumbnailDisplay </code>
+        </docsinternallink> large
+      </p>
+      <p>
+        in the event where <DocsInternalLink href="/KCard#prop:thumbnailDisplay">
+          <code> thumbnailDisplay</code>
+        </DocsInternalLink> and  <DocsInternalLink href="/KImg#prop:thumnailcaletype">
+          <code>
+            thumbnailScaleType
+          </code>
+        </DocsInternalLink> is not provided we will end something like this
+      </p>
+      <DocsShowCode language="html">
```

Comment:
I am thinking that's is the visual example below it- No?  I want to show few visual example.  Otherwise, the  documentation page will get cultured and I have not figured out a way to handle it.